extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -629,6 +646,43 @@ export class Bind {
   }
 
   /**
+   *
+   */
+  beginObservingElementForMutations_(element) {
+    // TODO(kmh287): What if parent is the body tag?
+    // TODO(kmh287): Generify logic for node observation strategy
+    // when bind supprots more dynamic nodes.
+    const elementToObserve = element.parentElement;
+    this.mutationObserver_.observe(elementToObserve, {childList: true});
+  }
+
+  /**
+   * Respond to observed mutations. Meant to be used with MutationObserver
```

Comment:
Nit: This comment is obvious given the method name. Instead, briefly describe what's happening in this method and why.